cpa/decision.go

```diff
@@ -18,6 +19,7 @@ type Violation struct {
 // Decision is a circleci flavoured output representing a policy decision.
 type Decision struct {
 	Status       Status      `json:"status"`
+	Cause        string      `json:"cause,omitempty"`
```

Comment:
Should cause be called "error_description" as this field will only be used for error state

Reply:
Or `Reason`